### PR TITLE
colors, width and height can be passed down throught props

### PR DESCRIPTION
**Pull request by ganmor**
Also there seem to be a problem with the position of the value when reducing the size.
I'll look into it if you are willing to accept prs.

### CODE REVIEW

File: src/react-gauge.js [hunk 1]
```diff
@@ -105,14 +116,13 @@ export default class ReactGauge extends React.Component {
 
   render() {
     let styles = this.getStyles();
-    console.log(styles.needlePath.d);
-
+		const viewBox = "0 0 " + this.state.width + ' ' + this.state.height;
     return(
-      <svg width={ this.state.width } height={ this.state.height }>
+      <svg width={ this.state.width } viewBox={viewBox} maxHheight={ this.state.height }>
```
Comment (migreva) [inline src/react-gauge.js]: What is this maxHheight property? I'm not seeing it in the Svg attribute list

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute

**Comment by migreva** [comment]
Definitely willing to accept PRs. This one looks good, just had a question about the svg attibute listed above

**Comment by ganmor** [comment]
Yes, I was thinking about using maxHeight since the gauge most certainly will be larger than higher but obvioulsy this would need to be provided as style={maxHeight}.
I updated the PR to use height instead.

Also the problem the position of the value was an error on my side so I think it's good.
Thanks.

**Comment by migreva** [comment]
Looks good.

**Comment by migreva** [comment]
This has been published to npm as react-gauge@0.0.3

**Comment by ganmor** [comment]
Great! thanks

**Pull request merged**